joke-app/src/app/api/jokes-api.ts

```diff
@@ -0,0 +1,26 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import Joke from "../types/Joke";
+
+export const jocksSlice = createApi({
+  reducerPath: "jocksSlice",
+
+  baseQuery: fetchBaseQuery({
+    baseUrl: "https://v2.jokeapi.dev/joke",
+  }),
+  tagTypes: [
+    'Jokes',
+  ],
+  endpoints: (builder) => ({
+    getAllJokes: builder.query<{jokes: Joke[]}, any>({
```

Comment:
We can add the proper type or remove it, form here 
my suggestion is to write it like 

getAllJokes: builder.query({
   query:  ()=>"/my-jokes"
})